src/lib.rs

```diff
@@ -59,7 +59,10 @@
     feature(const_ptr_offset_from)
 )]
 #![cfg_attr(feature = "unstable_const", feature(const_refs_to_cell))]
-#![cfg_attr(feature = "unstable_offset_of", feature(allow_internal_unstable))]
+#![cfg_attr(
+    all(feature = "unstable_offset_of", not(stable_offset_of)),
```

Comment:
IMO we should just remove the unstable feature. It's unstable so not covered by semver / stability.